### DISCUSSION TITLE
Plotting entries, exits, indicators, etc

### DISCUSSION THREAD

**Discussion by kaustv-datta** [nautechsystems/nautilus_trader#1187] [Q&A] [answered]
Hi Team,

Is there a guide on how to plot things like entry/exits signals and indicators for a single backtest run?

My use cases -
1. Visually check whether a custom indicator looks like it is supposed to.
2. The strategy behaves as it is supposed to with the entry and exit logic.

P.S. Amazing work guys! I'm super excited to develop my first strategy using this framework!

**Comment by kaustv-datta** [comment]
@cjdsellers any thoughts on this?

**Reply by limx0** [reply, marked as answer]
Hey @kaustv-datta - the streaming config allows dumping out feather files which can be read by pandas (and then your choice of plotting library)
